docs/csharp/programming-guide/concepts/linq/linq-to-xml-overview.md

```diff
@@ -24,26 +24,43 @@ XML has been widely adopted as a way to format data in many contexts. For exampl
   
  For example, you might have a typical XML purchase order as described in [Sample XML File: Typical Purchase Order (LINQ to XML)](sample-xml-file-typical-purchase-order-linq-to-xml-1.md). By using [!INCLUDE[sqltecxlinq](~/includes/sqltecxlinq-md.md)], you could run the following query to obtain the part number attribute value for every item element in the purchase order:  
   
-```csharp  
-IEnumerable<string> partNos =  
-from item in purchaseOrder.Descendants("Item")  
-select (string) item.Attribute("PartNumber");  
-```  
-  
+```csharp
+// Load the XML file containing the purchase orders
+string folderPath = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
```

Comment:
This won't work on .NET Core on Mac or Linux. We should update this to load a file from the current directory, or parse a constant string declared in the sample.

Reply:
I'm afraid I don't have the means to try this out on MacOS, but it looks like getting the file from the current directory will work for our crossplatform needs:
![chrome_2018-10-30_00-40-54](https://user-images.githubusercontent.com/8032019/47696316-9beba580-dbdc-11e8-85b5-fc7bebe458be.png)
I've done the majority of my development on a Windows machine for both work and leisure, so I haven't had much opportunity to play with .NET Core on other operating systems.

Reply:
I like the idea of using the current directory. That works great.